src/index.ts

```diff
@@ -358,9 +363,10 @@ async function main() {
 
   if (!command) {
     console.error(
-      'Usage: vibe-tools [--model=<model>] [--max-tokens=<number>] [--from-github=<github_url>] [--output=<filepath>] [--save-to=<filepath>] [--hint=<hint>] <command> "<query>"\n' +
+      'Usage: vibe-tools <command> "<query>" [--provider=<provider>] [--model=<model>] [--save-to=<filepath>] [--from-github=<github_url>] [--quiet] [--debug] [...other command-specific parameters]\n' +
         '       Note: Options can be specified in kebab-case (--max-tokens) or camelCase (--maxTokens)\n' +
-        '       Both --key=value and --key value formats are supported'
+        '       Both --key=value and --key value formats are supported\n' +
+        "       Run 'vibe-tools <command> --help' for command-specific options"
```

Comment:
```suggestion
        '       Both --key=value and --key value formats are supported\n'
```
I don't think that --help is implemented for commands